screens/home.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import { View, ScrollView, Dimensions } from 'react-native'
+import { RFValue } from 'react-native-responsive-fontsize'
+import BasicPage from '../components/page-template'
+import i18n from '../lib/i18n'
+import Backend from '../lib/backend'
+import TeamItem from '../components/team-item'
+import { Text, List, ActivityIndicator } from 'react-native-paper'
+import { useEffect } from 'react'
+import { useState } from 'react'
+import { getUserGivenName } from '../lib/auth'
+
+export default function HomeScreen({ route, navigation }) {
+  const { authToken } = route.params
+  const [events, setEvents] = useState(null)
+
+  useEffect(() => {
+    Backend.fetchTeams(route.params.authToken).then(teams => {
+      const events = teams.reduce((obj, team) => {
+        if (!obj[team.event.name]) obj[team.event.name] = []
+        obj[team.event.name].push(team)
+        return obj
+      }, {})
+      setEvents(events)
+    })
+  }, [])
+
+  const width = Dimensions.get('window').width
+
+  const handleSelectTeam = item => navigation.navigate('PRE_INST', { item, authToken })
+
+  return (
+    <BasicPage>
+      {!events ? (
+        <View style={styles.loading}>
+          <ActivityIndicator size="large" />
+        </View>
+      ) : (
+        <>
+          <View style={styles.header}>
+            <Text style={styles.user}>
+              {i18n.t('hello_user', { name: getUserGivenName(authToken) })}
+            </Text>
+          </View>
+          <ScrollView style={{ width }}>
+            {Object.keys(events).map(event => (
```

Comment:
The 'key' of the object is not the event, it's the `eventKey` or the `eventId` or something. Maybe use `Object.entries` instead.